ui/src/app/app.component.ts

```diff
@@ -60,18 +59,28 @@ export class AppComponent  implements OnInit, OnDestroy {
                 this.isConnected = true;
                 this.startWarningWorker();
             }
+            this.startVersionWorker();
         });
     }
 
+    stopVersionWorker(): void {
```

Comment:
Why not use a more generic function that take in parameter the name of the worker in order to have just one function and don't maintain 2 different functions that make the same things ?